Editor/Scenes/TopMenu/TopMenu.gd

```diff
@@ -0,0 +1,53 @@
+extends HBoxContainer
+
+var menus = {
+	"File": {
+		"node": "FileMenu",
+		"elements": { "Open": "_open", "Save": "", "Save as...": "", "New": "", "Separator": "", "Quit": "" }
+	},
+	"Edit": {
+		"node": "EditMenu",
+		"elements": { }
+	},
+	"View": {
+		"node": "ViewMenu",
+		"elements": {  }
+	},
+	"Help": {
+		"node": "HelpMenu",
+		"elements": {  }
+	}
+}
+onready var file_menu = $FileMenu
+onready var edit_menu = $EditMenu
+onready var view_menu = $ViewMenu
+onready var help_menu = $HelpMenu
```

Comment:
These are likely unnecessary since we already have the node name in the dictionary. Even better is to replace the node names in the dict with these references :)

Reply:
oh crap I forgot to delete them xD